components/eamxx/src/physics/p3/eamxx_p3_process_interface.cpp

```diff
@@ -204,6 +205,15 @@ void P3Microphysics::initialize_impl (const RunType /* run_type */)
 {
   // Gather runtime options
   runtime_options.max_total_ni = m_params.get<double>("max_total_ni");
+
+  m_p3constants.set_p3_defaults();
+
+  //does this work when the value is not in the nl?
+  m_p3constants.p3_autoconversion_factor = m_params.get<double>("p3_autoconversion_factor");
+
+//temp to check   
+std::cout << "AAAAAAAAAAAAAANG "<<m_p3constants.p3_autoconversion_factor <<"\n" << std::flush;
```

Comment:
When you are done with debugging, we can remove this line.

Reply:
Or actually, we could keep it, but make it better. You could do something like this
```
  m_atm_logger->info("P3 Constants:");
  m_atm_logger->info("  p3 autoconversion factor: " + std::to_string(m_p3constants.p3_autoconversion_factor));
  // same for other params
```
This way we can always verify that the params were correct by looking at the atm log file.